Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -632,7 +632,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
          _("Removes a variable at the specified index of an object array "
            "variable."),
          _("Remove variable at index _PARAM2_ from array variable _PARAM1_ of "
-           "object _PARAM0_"),
```

Comment:
I've updated this because the object is already giving the context (and fixing the fact that we put "object _PARAM0_", while we don't use "object" in sentences, as it's implied by the object name itself.

Reply:
Oh yeah, this context wasn't visible at first glance unless we know it or we check in the application, my bad.
![image](https://user-images.githubusercontent.com/1670670/120553721-1dd98600-c3f9-11eb-8231-10c5b687ffd3.png)